samples/super-quickstart/SuperQuickStart-OMD.sh

```diff
@@ -0,0 +1,448 @@
+bin/helm repo add jetstack https://charts.jetstack.io
+bin/helm repo add hashicorp https://helm.releases.hashicorp.com
+bin/helm repo add fybrik-charts https://fybrik.github.io/charts
+bin/helm repo update
+
+header "\nInstall Cert-manager"
+bin/helm install cert-manager jetstack/cert-manager \
+    --namespace cert-manager \
+    --version v${CERT_MANAGER_VERSION} \
+    --create-namespace \
+    --set installCRDs=true \
+    --wait --timeout 600s
+
+header "\nInstall Vault"
+bin/helm install vault fybrik-charts/vault --create-namespace -n fybrik-system \
+    --set "vault.injector.enabled=false" \
+    --set "vault.server.dev.enabled=true" \
+    --values https://raw.githubusercontent.com/fybrik/fybrik/v${FYBRIK_VERSION}/charts/vault/env/dev/vault-single-cluster-values.yaml
+bin/kubectl wait --for=condition=ready --all pod -n fybrik-system --timeout=120s
+
+# If using openshift:
+# bin/helm install vault fybrik-charts/vault --create-namespace -n fybrik-system \
+#     --set "vault.global.openshift=true" \
+#     --set "vault.injector.enabled=false" \
+#     --set "vault.server.dev.enabled=true" \
+#     --values https://raw.githubusercontent.com/fybrik/fybrik/v${FYBRIK_VERSION}/charts/vault/env/dev/vault-single-cluster-values.yaml
+# bin/kubectl wait --for=condition=ready --all pod -n fybrik-system --timeout=120s
+
+header "\nInstall OpenMetaData"
+export FYBRIK_BRANCH=v1.2.0
+curl https://raw.githubusercontent.com/fybrik/fybrik/v1.2.0/third_party/openmetadata/install_OM.sh | bash -
+
+
+
+header "\nInstall control plane"
+
+bin/helm install fybrik-crd fybrik-charts/fybrik-crd -n fybrik-system --version ${FYBRIK_VERSION} --wait
+
+bin/helm install fybrik fybrik-charts/fybrik --set coordinator.catalog=openmetadata --set openmetadataConnector.openmetadata_endpoint=http://openmetadata.open-metadata:8585/api -n fybrik-system --version ${FYBRIK_VERSION} --wait # --values=../values.yaml --wait
+sleep 5
+
+header "\nInstall module"
+if [[ ! -z ${MODULE_PATH} ]]; then
+    bin/kubectl apply -f ${MODULE_PATH} -n fybrik-system
+else
+    bin/kubectl apply -f https://github.com/fybrik/${MODULE}/releases/latest/download/module.yaml -n fybrik-system
+fi
+
+sleep 5
+
+# END OF PART 1 OF THE QUICKSTART
+# STARTING READ SAMPLE:
+
+header "\nCreate notebook sample"
+bin/kubectl create namespace fybrik-notebook-sample
+bin/kubectl config set-context --current --namespace=fybrik-notebook-sample
+
+header "\nInstall localstack"
+bin/helm repo add localstack-charts https://localstack.github.io/helm-charts
+bin/helm install localstack localstack-charts/localstack --set startServices="s3" --set service.type=ClusterIP
+bin/kubectl wait --for=condition=ready --all pod -n fybrik-notebook-sample --timeout=120s
+bin/kubectl port-forward svc/localstack 4566:4566 &
+
+header "\nUpload sample dataset to localstack"
+curl -L https://raw.githubusercontent.com/fybrik/fybrik/master/samples/notebook/PS_20174392719_1491204439457_log.csv -o PS_20174392719_1491204439457_log.csv
```

Comment:
The file name is used many times along the script - use a variable instead

Reply:
changed to sample.csv